src/aioamazondevices/implementation/sequence.py

```
@@ -26,18 +31,88 @@ def __init__(
-    ) -> None:
-        """Send message to specific device."""
+    ) -> dict[str, Any]:
+        """Convert message to operation node."""
         if not self._session_state_data.login_stored_data:
             _LOGGER.warning("No login data available, cannot send message")
-            return
+            raise CannotConnect(
+                "Cannot perform %s action on %s as not logged in",
+                message_type,
+                device.serial_number,
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)